nano/node/json_handler.cpp

```diff
@@ -2069,19 +2069,37 @@ void nano::json_handler::database_txn_tracker ()
 void nano::json_handler::delegators ()
 {
 	auto account (account_impl ());
+	auto count (count_optional_impl ());
+	auto threshold (threshold_optional_impl ());
+	auto head_str (request.get_optional<std::string> ("head"));
+
+	nano::account head_account (0);
+	if (head_str)
+	{
+		auto error = head_account.decode_account (*head_str);
+		if (error)
+		{
+			ec = nano::error_common::bad_account_number;
+		}
+	}
 	if (!ec)
 	{
-		boost::property_tree::ptree delegators;
 		auto transaction (node.store.tx_begin_read ());
-		for (auto i (node.store.account.begin (transaction)), n (node.store.account.end ()); i != n; ++i)
+		boost::property_tree::ptree delegators;
+		uint64_t delegators_count = 0;
```

Comment:
This variable seems assigned but otherwise unused.

Reply:
True, let me remove it.